docs/UserGuide.md

```diff
@@ -202,11 +198,14 @@ Imports all persons and their details from a CSV file of your specification.
 
 * The file path should be **absolute**.
 * This command will only import persons' particulars. To import exam scores, take a look at [`importExamScores`](#importing-exam-scores-from-a-csv-file--importexamscores).
+* All compulsory parameters **must** be present in the column headings (i.e. the first row) of the CSV file you are importing i.e. `name`, `email`, `phone` and `address`.
```

Comment:
This description is clear and will ensure users understand how to use the import command